docs/evm/clients/web3-js.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Web3.js
+title: Web3.js on Flow Blockchain
 sidebar_label: Web3.js
 sidebar_position: 2
 ---
 
-# Web3.js
+# Web3.js on Flow Blockchain
```

Comment:
The thoughts here were that it would help with SEO, I'd imagine that people interested in this guide would google something along the lines of: 'Web3js Flow' or 'Web3js on Flow'. The titles play a big role in this.

Reply:
understood, but from a client perspective, doesn't quite make sense